ui/src/state/mod.rs

```diff
@@ -363,13 +363,11 @@ impl State {
             .cloned()
     }
 
-    pub fn get_without_me(&self, identities: Vec<Identity>) -> Vec<Identity> {
-        let mut set = HashSet::new();
-        set.insert(&self.account.identity);
-
+    pub fn get_without_me(&self, identities: &[Identity]) -> Vec<Identity> {
```

Comment:
I changed this from a `&Vec` to a `&[T]` because I read [here](https://rust-unofficial.github.io/patterns/idioms/deref.html) that this is better